app/scripts/lib/channels/notifier.js

```diff
@@ -18,9 +18,10 @@ define(function (require, exports, module) {
   var EVENTS = {
     COMPLETE_RESET_PASSWORD_TAB_OPEN: 'fxaccounts:complete_reset_password_tab_open',
     DELETE: 'fxaccounts:delete',
+    DEVICE_NAME_CHANGE: 'fxaccounts:device_name_change',
     PROFILE_CHANGE: 'profile:change',
     SIGNED_IN: 'internal:signed_in',
-    SIGNED_OUT: 'internal:signed_out'
+    SIGNED_OUT: 'fxaccounts:logout'
```

Comment:
> The reason I say possibly is because of this `!localOnly` guard around the destroy call...

My local branch is working again now and the duplicate call is indeed being made. Will fix.